### ISSUE TITLE
[Bug] Unsupported function torch.softmax

### ISSUE THREAD

**Issue by soodoshll** [hidet-org/hidet#98]
It seems that PyTorch 2.0 adds several functions to the top namespace (torch.**), and `torch.softmax` is one of them. (https://pytorch.org/docs/master/torch.html) It's equivalent to `torch.nn.functional.softmax`. However, Hidet now can't recognize this alias.

Maybe it's also worth checking if there are other function aliases not included.

**Comment by yaoyaoding** [comment]
Yes, currently I add those torch->hidet operator mappings when I run some new model and it requires those operators. We still lack hands to add these operator mappings (both for pytorch and onnx frontend), and operator itself.
